docs/running-on-kubernetes.md

```diff
@@ -200,42 +200,88 @@ from the other deployment modes. See the [configuration page](configuration.html
   </td>
 </tr>
 <tr>
-  <td><code>spark.kubernetes.submit.caCertFile</code></td>
+  <td><code>spark.kubernetes.authentication.submission.caCertFile</code></td>
```

Comment:
They use `spark.authenticate` in other places - but not sure if we should be following that precedent: https://spark.apache.org/docs/latest/configuration.html